### PR TITLE
fix #78080 akhisarhaber.com

### LINKED ISSUE #78080: www.akhisarhaber.com
### Issue URL (Ads)
 [http://www.akhisarhaber.com/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=http%3A%2F%2Fwww.akhisarhaber.com%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | uBlock Origin v1.34.0
Browser: | Chrome
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base, <br/>EasyList<br/><br/><b>Language-specific:</b><br/> AdGuard Turkish

### PR DESCRIPTION
**Pull request by wout-codes**
#78080

### CODE REVIEW

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]
twitter widget is missed

File: TurkishFilter/sections/specific.txt [hunk 1]
```diff
@@ -42,6 +42,9 @@ yemektarifleri.org,lifebursa.com,mshowto.org,macizlee1.com,macizlee.ne,dunyaatla
 fullhdfilmcibaba.com,720pizlemek.com,1080pfilmizle.net,fullhdfilmcidayi.com,hdfilmizledayi.com,720-pizleme.com,fullhdfilmizlebaba.com,evrenselfilmlerim.net###adStarter
 !
 aksarayhaber68.com##.Reklam_001
+akhisarhaber.com,detaykibris.com,ajans32.com##.index_banner
+akhisarhaber.com##.banner_code
+||akhisarhaber.net/reklam2/
```
Comment (Sergey-Lyapin) [inline TurkishFilter/sections/specific.txt:47]: we already have `||akhisarhaber.net/reklam*/` in the turkish filter

File: TurkishFilter/sections/specific.txt [hunk 2]
```diff
@@ -42,6 +42,9 @@ yemektarifleri.org,lifebursa.com,mshowto.org,macizlee1.com,macizlee.ne,dunyaatla
 fullhdfilmcibaba.com,720pizlemek.com,1080pfilmizle.net,fullhdfilmcidayi.com,hdfilmizledayi.com,720-pizleme.com,fullhdfilmizlebaba.com,evrenselfilmlerim.net###adStarter
 !
 aksarayhaber68.com##.Reklam_001
+akhisarhaber.com,detaykibris.com,ajans32.com##.index_banner
+akhisarhaber.com##.banner_code
```
Comment (Sergey-Lyapin) [inline TurkishFilter/sections/specific.txt:46]: this rule changes nothing

File: SocialFilter/sections/specific.txt [hunk 3]
```diff
@@ -3,7 +3,7 @@
...
...ilicon.com,rottentomatoes.com,rtcc.org,sabah.com.tr,sat1gold.de,scalescale.com,searchmarketingexpo.com,seattletimes.com,sectionstudios.com,shacknews.com,shefinds.com,sheknows.com,sofarsogood.fr,softexia.com,softonic.com,solidfiles.com,space.com,sparknotes.com,spectator.org,spin.com,sucuri.net,t3.com,tablet-news.com,talkingpointsmemo.com,talkweb.eu,tarzgo.com,techeblog.com,techpowerup.com,techpp.com,techradar.com,techspot.com,telecomstechnews.com,theblemish.com,thebolditalic.com,thedissolve.com,theguardian.com,theonion.com,thestar.com,thestudenthousingcompany.com,ti.com,tickld.com,timesofindia.indiatimes.com,tmonews.com,tnr.com,trendhunter.com,troyhunt.com,tubestart.com,tunein.com,tvline.com,typomaps.net,u.tv,ucl.ac.uk,udreview.com,uk-mkivs.net,ulke.com.tr,urbanomnibus.net,usatoday.com,vg247.com,vice.com,victoriassecret.com,viralnova.com,virginmedia.com,vulture.com,wallpaper.com,washingtontimes.com,wbs-law.de,wearabletechnology-news.com,webref.ru,webs.com,weeklystandard.com,wegotthiscovered.com,wetpaint.com,whatculture.com,whowhatwear.com,windowsvalley.com,wn.com,woot.com,wunderground.com,x17online.com,xconomy.com,xsreviews.co.uk,yachtturkiye.com,znak.com,lxax.com##.social:not(body)
```
Comment (Sergey-Lyapin) [inline SocialFilter/sections/specific.txt:6]: this rule hides links to their own socials, I meant twitter widget in the right sidebar

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]

File: SocialFilter/sections/specific.txt (hunk 3 above)
Comment (wout-codes) [reply SocialFilter/sections/specific.txt:6]: What widget, be clear or do not even say anything.
image

Comment (Sergey-Lyapin) [reply SocialFilter/sections/specific.txt:6]: Scroll down
Screenshot 2021-03-23 at 17 22 07

File: SocialFilter/sections/specific.txt [hunk 4]
```diff
@@ -39,6 +39,7 @@ giantbomb.com,strefainwestorow.pl,narcity.com,lingohut.com,mtlblog.com,tesisatma
...
 !
 gifyu.com##.panel-share-networks
+akhisarhaber.com###twitter-widget-0
```
Comment (Sergey-Lyapin) [inline SocialFilter/sections/specific.txt:42]: For such cases it's better to use 2 rules under this comment:
`! ##.twitter-timeline + widgets.js`
Pleass add akhisarhaber.com to each of them

**Pull request merged**